### ISSUE TITLE
Why '/opme' step?

### ISSUE THREAD

**Issue by SirFrostingham** [SchnitzelPommes/DayZ-SA-Tomato#32]
Your tool is the best. Why do we need this step?
 - Press "T" again and this time type /opme and press "enter"

Is it a security measure to prevent some hack? Was the Admins.txt Steam64ID method failing or not working?

This step to me seems over the top, unless it is for anti-hack. It will broadcast the admin message to the whole server. Another issue is /opme is incompatible with the Vanilla++ mod if the server uses the admin feature. I edited the Vanilla++ code to add a case to the switch statement to allow /opme to work on my server.

Thank you for this tool!

**Comment by SchnitzelPommes** [comment]
You only need it the first time to set urself as admin

**Comment by SirFrostingham** [comment]
I saw that. It solves a security need, I take it? Whatever the case, I provided the feedback which was my goal. Thanks again for the best admin tools!

**Comment by SchnitzelPommes** [comment]
no its not about security its for easy setup so ppl dont have to edit files

**Comment by SirFrostingham** [comment]
What I liked about the previous method was I could add a Steam64ID to the list and not need to give out my server's password. This was a nice feature, since sometimes you are dealing with new admins that you may not want to give that password to.

**Comment by SirFrostingham** [comment]
Can some admin I decided to trust give out the admin password to his friends and explain how to use the Tomato Admin tools and add themselves to the list? The way DayZ / Arma tools work, I think anyone that has the password can log in as admin. Maybe BEC.exe can stop them, but that would require testing by removing my GUID from the approved list and try to login / add myself to use the Tomato admin tools. Still, I do not want to give out my server's password. 

My server was a very high ranked server to the 20s rank, but is now 50th rank due to base persistence bug with DayZ. Anyway, there is a player community there and some are admins, but I don't want them to auto add themselves to a list or have access to my server password.

**Comment by SchnitzelPommes** [comment]
okay i make something clear here
1. #login PW and /opme works only once !
2. if u want to add another admin you can go to profilesfolder/Config/player and there will be a player file for every player ever on your server u ask ur admin for his steam64id like before then u open the file which is named after the steam64 id of players and set Admin = 1 (player should be offline meanwhile)

thats how u add admin /opme is only for easy setup so if some1 inexperienced trying to use that tool its easy as it can get before u had to restart the server to add a new admin now u can do it on the fly since the player file will load on login and save on logout (thats why the player should be offline when u edit the file it will reset on logout)

**Comment by SirFrostingham** [comment]
One note for you. While testing between 2 different systems today: 1) desktop and 2) laptop, I had to use /opme on the laptop a second time after a reboot.

Process:
1. Laptop: login as admin -> /opme -> logout/login = Access to tools
2. PC: login as admin -> /opme -> logout/login = Access to tools
3. Go back to laptop from step # 1 -> reboot -> run game -> try to press M -> no access to tools -> login as admin -> /opme -> logout/login = Access to tools again
4. *just now while writing this up* Go back to PC from step # 2: run game -> try to press M -> no access to tools -> login as admin -> /opme -> logout/login = Access to tools again

Something knocks the user off of the list between 2 systems.

**Comment by SchnitzelPommes** [comment]
same steam account?
are u at discord? pm if if so

**Comment by SirFrostingham** [comment]
Same Steam account. I'll find your Discord.
